ast.grace

```diff
@@ -2300,8 +2300,8 @@ def bindNode is public = object {
     var value is public := val'
 
     method end -> Position { value.end }
-    method nameString { value ++ ":=(1)" }
-    method canonicalName { value ++ ":=(_)" }
+    method nameString { dest.asString ++ ":=(1)" }
+    method canonicalName { dest.asString ++ ":=(_)" }
```

Comment:
These don't seem right to me.  I think that dest will be an identifier node, so dest.asString will be something like `"identifier‹foo›"`.   Perhaps this should be `identifier.nameString`?